code/__HELPERS/text.dm

```diff
@@ -47,9 +47,15 @@
 /proc/sanitize_filename(t)
 	return sanitize_simple(t, list("\n"="", "\t"="", "/"="", "\\"="", "?"="", "%"="", "*"="", ":"="", "|"="", "\""="", "<"="", ">"=""))
 
+///proc to check whether a string contains anything in the OOC in IC filter, returns true if there is such a filter and it does
+/proc/contains_ic_filtered_words(t)

```

Comment:
why is this a proc :(

Reply:
because, as above, it's very likely that there will end up being other things that need the IC filter check, and why reuse the code

Reply:
I think you're looking for a define then.